org.kuchelmeister.ToolboxTuner.json

```diff
@@ -0,0 +1,61 @@
+{
+    "id" : "org.kuchelmeister.ToolboxTuner.Devel",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "46",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable",
+        "org.freedesktop.Sdk.Extension.llvm15"
+    ],
+    "command" : "toolbox-tuner",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--talk-name=org.freedesktop.Flatpak"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin",
+        "build-args" : []
+    },
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "toolbox-tuner",
+            "builddir" : true,
+            "buildsystem": "meson",
+            "build-options": {
+                "env": {
+                    "CARGO_HOME": "/run/build/toolbox-tuner/cargo"
+                }
+            },
+            "config-opts": [
+                "-Dprofile=flathub"
+            ],
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://github.com/13hannes11/toolbox-tuner/archive/refs/tags/dev.tar.gz",
```

Comment:
"dev" as a tag. I guess there is no release yet?

Reply:
I was planning to have the release and review in parallel. in principle the software is ready for release but it might be that this review surfaces some problems that need fixing. Thus, I was planning to not yet have a full release build.

Reply:
The version v0.1.0 is now published